tests/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
You can use the base package.json file. There is no need to create a separate package.json for tests/.

Reply:
since common js is used in the tests/, separate package.json is used for that